package.json

```diff
@@ -52,12 +57,22 @@
         "eslint-plugin-import": "^2.18.0",
         "eslint-plugin-jsdoc": "^15.5.3",
         "eslint-plugin-prettier": "^3.1.0",
+        "eslint-plugin-scanjs-rules": "^0.2.1",
+        "eslint-plugin-security": "^1.4.0",
         "jest": "^24.8.0",
         "prettier": "^1.18.2",
         "rollup": "^1.16.7",
         "rollup-plugin-babel": "^4.3.3",
         "rollup-plugin-commonjs": "^10.0.1",
         "rollup-plugin-node-resolve": "^5.2.0",
         "rollup-plugin-terser": "^5.1.1"
+    },
+    "peerDependencies": {
```

Comment:
Some parts of `eslint`-related dependencies are duplicated across different types of dependency sections. Is that intentional? 

We need to have clear understanding if and what cloudify-ui-common user should install it (`peerDependencies`) or will it already be installed (`dependencies`).

Reply:
Yes, this is intentional - `dependencies` and `peerDependencies` are configured correctly according to their intended usage